src/registrar/models/utility/admin_form_order_helper.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from typing import Dict
+from django.forms import ModelChoiceField
+
+logger = logging.getLogger(__name__)
+
+
+class SortingDictInterface:
```

Comment:
Inspired by Typescript/Angular patterns?

Reply:
Yes! I can remove the 'interface' portion if necessary. I left it in as it was more descriptive as to its purpose, but I don't believe that name is entirely accurate